PUBLISHING.md

```diff
@@ -6,14 +6,11 @@ For now it has to be done manually.
 
 ## Release commit
 
-- Update `CHANGELOG.md` (replace `Unpublished` with the version and the release date).
-- Bump version in `umbral-pre/Cargo.toml`.
-- Bump version in `umbral-pre-python/Cargo.toml`.
-- Bump version in `umbral-pre-python/docs/conf.py`.
-- Bump version in `umbral-pre-python/setup.py`.
-- Bump version in `umbral-pre-wasm/Cargo.toml`.
-- Bump version in `umbral-pre-wasm/package.template.json`.
-- Tag the release commit with the version tag (in `v*.*.*` format).
+- Update `CHANGELOG.md` (replace `Unreleased` with the version and the release date).
+- Use Python [Bumpversion](https://github.com/c4urself/bump2version/) to autmoatically update relevant version strings throughout the repo.
+  - `bump2version minor --current-version 0.3.3`
```

Comment:
Seems to be an extra space here.
Also maybe remove an explicit version and phrase it something like `bump2version minor --current-version <major>.<minor>.<patch>`